src/test/scala/org/apache/spark/deploy/k8s/integrationtest/KubernetesSuite.scala

```diff
@@ -20,6 +20,8 @@ import java.io.File
 import java.nio.file.{Path, Paths}
 import java.util.UUID
 import java.util.regex.Pattern
+import java.sql.DriverManager
+import org.apache.hive.jdbc.HiveDriver
```

Comment:
There should be an empty line separating java imports and the rest, and the order of imports should be java, scala, other third-party, and finally spark. So `org.apache.hive.jdbc.HiveDriver` should be to after `io.fabric8...`.